Makefile

```diff
@@ -28,6 +31,9 @@ prod:
 worker:
 	celery -A genomic_data_service.region_indexer_task worker --loglevel=INFO
 
+worker_docker:
+	DOCKER=worker_docker celery -A genomic_data_service.region_indexer_task worker --loglevel=INFO
```

Comment:
This is most likely the best place to pass the config to celery whether it is run with docker or not.